ingester/mainloop.go

```diff
@@ -166,7 +166,10 @@ func (i *ingester) SendBlocks(ctx context.Context, blocksCh <-chan models.RPCBlo
 	i.log.Info("SendBlocks: Starting to receive blocks")
 	blocks := make(map[int64]models.RPCBlock) // Buffer for temporarily storing blocks that have arrived out of order
```

Comment:
suggest "outOfOrderBlocks" or "collectedBlocks" (just for more readability)
```suggestion
	collectedBlocks := make(map[int64]models.RPCBlock) // Buffer for temporarily storing blocks that have arrived out of order
```

Reply:
Good idea